cautious.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cautious
+spec:
+  shortDescription: Cautiously operate inside clusters
+  homepage: https://github.com/fedeztk/kubectl-cautious
+  description: |
+    Cautiously run kubectl commands, no more accidental deletions!
+    Supports regexes and is configured via a yaml file under ~/.kube
+  version: v0.0.3
+  platforms:
+  - selector:
+      matchExpressions:
+      - key: os
+        operator: In
+        values:
+        - darwin
+        - linux
```

Comment:
Isn't this a Go program? How do you use the same binary for both macOS and Linux? I'm pointing this out since the tests don't execute the plugin to validate if it runs.

Reply:
Thanks, I was using the wrong `yaml` file. I updated the PR with the correct one.